txnkv/txnlock/lock_resolver.go

```diff
@@ -238,11 +238,27 @@ func (lr *LockResolver) BatchResolveLocks(bo *retry.Backoffer, locks []*Lock, lo
 	txnInfos := make(map[uint64]uint64)
 	startTime := time.Now()
 	for _, l := range expiredLocks {
+		logutil.Logger(bo.GetCtx()).Info("BatchResolveLocks handling lock", zap.Stringer("lock", l))
```

Comment:
Should we use `logutil.Debug(...)`?

Reply:
Ah sorry. I added this when debugging and forgot to remove it. Keep it as a debug-level log might be a good idea.